<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Amar" ignored="0" name="Test" tests="2" failures="2" timestamp="2020-08-03T23:48:01 IST" time="0.237" errors="0">
  <testcase name="TC02" time="0.028" classname="automation.RahulFlightReservation">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at automation.RahulFlightReservation.TC02(RahulFlightReservation.java:52)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TC02 -->
  <testcase name="TC02" time="0.09" classname="automation.RahulFlightReservation">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Utils.Base.takeScreenshot(Base.java:27)
at Utils.Listener.onTestFailure(Listener.java:57)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- TC02 -->
</testsuite> <!-- Test -->
